### DISCUSSION TITLE
Notification Test

### DISCUSSION THREAD

**Discussion by ghost** [amzn/selling-partner-api-models#3150] [General]
I would like to test that my application is able to handle SQS notifications correctly. Example, receiving new orders notification, parsing them and calling other order API method with the ids in the notification. I couldn't find the doc about it.

Is there a way to subscribe to some test sqs notifications and validate that my application is working or do I need to test it with real orders with a selling partner ? 

Thanks

**Comment by GJ-JG** [comment]
Hi @mlefebvre-deblecx , it will be better to test with the real orders on your application if you are referring to ORDER_STATUS_CHANGE notification type.
